APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud

```diff
@@ -0,0 +1,14 @@
+IME Fraud API submission
+
+| **Field** | Description |
+| ---- | ----- |
+| API name | IMEI Fraud|
+| API owner | MTN - Waseem Amra |
+| Initial API Contributors | MTN - Waseem Amra | 
+| API summary | This API allows for the API Consumer to understand if fraud has been associated to the device. Fraud can range from lost, suspended, stolen, blacklisted. |
```

Comment:
```suggestion
| API summary | This API allows for the API Consumer to understand if any fraud markers have been associated with the device. Fraud markers may be applied if the device is lost, suspended, stolen, blacklisted, etc. |
```

Reply:
Accepted

Reply:
@wwAMRA if you are ok with this proposed change, can you please commit it into the PR?